imageregistry/v1/01-crd.yaml

```diff
@@ -619,14 +619,13 @@ spec:
               format: int32
             keepTagRevisions:
               description: keepTagRevisions specifies the number of image revisions
-                for a tag in an image stream that will be preserved. Defaults to 5.
+                for a tag in an image stream that will be preserved. Defaults to 3.
               type: integer
             keepYoungerThan:
               description: keepYoungerThan specifies the minimum age of an image and
                 its referrers for it to be considered a candidate for pruning. Defaults
-                to 96h (96 hours).
-              type: integer
-              format: int64
+                to 60m (60 minutes).
+              type: string
```

Comment:
this means that old integers won't validate anymore. There is `x-kubernetes-int-or-string: true`, usually used with the `IntOrString` type, and afaik not supported by controller-tools via annotation. But you can patch it in by using a patch file next to 01-crd.yaml here, compare https://github.com/openshift/api/blob/master/operator/v1/0000_20_kube-apiserver-operator_01_config.crd.yaml-merge-patch.

Reply:
This fields is ment to be populated by user, and our documentation and godoc give examples like "96h" or "60m". The meaning of this field as an integer is not defined.